src/extension-test/unit/cursor-doc/paredit-test.ts

```diff
@@ -1268,6 +1268,23 @@ describe('paredit', () => {
         await paredit.backspace(a);
         expect(textAndSelection(a)).toEqual(textAndSelection(b));
       });
+      it('Deletes whitespace to the left of the cursor', async () => {
+        const a = docFromTextNotation(
+          `
+(defn nil-if-blank [s]
+  (if (str/blank? s) 
+    |nil s))
+        `.trim()
+        );
+        const b = docFromTextNotation(
+          `
+(defn nil-if-blank [s]
+  (if (str/blank? s) |nil s))
+        `.trim()
+        );
+        await paredit.backspace(a);
+        expect(textAndSelection(a)).toEqual(textAndSelection(b));
+      });
```

Comment:
I think we should cover more of the cases with tests. So that it is clear how it should behave in the different situations we've discussed.

We usually replace newlines with that dot-character. Maybe writing the text like this with literal newlines is better, though, so let's keep it.

However, we can use much less text, I think. It doesn't need to be real code.

Reply:
Ok, I'll work on adding more tests and using less text next time I get a chance to work on this.